keras/backend/cntk_backend.py

```diff
@@ -232,8 +238,11 @@ def placeholder(
     cntk_shape = tuple(cntk_shape)
 
     if dynamic_axis_num > len(cntk_shape):
-        raise ValueError('CNTK backend: the input dim '
-                         'is less than the number of dynamic axis')
+        raise ValueError('CNTK backend: request to create placeholder'
+                         'with %d dimension, which is less than the '
+                         '%d dynamic axis request. Please increase'
+                         'the input dimension or reduce the dynamic axis'
+                         % (len(cntk_shape, dynamic_axis_num)))
```

Comment:
the dynamic axis is cntk concept... let me try to update the message with more standard information.